docs/shared/packages/playwright/playwright-plugin.md

```diff
@@ -65,6 +65,24 @@ The `@nx/playwright/plugin` is configured in the `plugins` array in `nx.json`.
 
 The `targetName` and `ciTargetName` options control the name of the inferred Playwright tasks. The default names are `e2e` and `e2e-ci`.
 
+### Splitting E2E Tests
```

Comment:
Unrelated to this change, but the Playwright overview is formatted wrong. The `Nx 18+` tab contains all the inferred and task splitting content as well -- it should just be the `nx add @nx/playwright` codeblock.

<img width="1032" alt="Screenshot 2024-08-16 at 10 33 57 AM" src="https://github.com/user-attachments/assets/0a72a901-b9a4-4da1-9d54-d4efa255eb98">

Reply:
Inferred and task splitting doesn't work in Nx < 18